src/Modules/Windows-Full/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.psd1

```diff
@@ -8,7 +8,6 @@ PowerShellVersion="3.0"
 CLRVersion="4.0"
 NestedModules="Microsoft.PowerShell.Commands.Management.dll"
 HelpInfoURI = 'https://go.microsoft.com/fwlink/?linkid=390785'
-AliasesToExport = @("gcb", "scb", "gin", "gtz", "stz")
```

Comment:
The same - we now control all the aliases in the tests and can safely export all aliases by default.